pkg/route/api/v1/conversion.go

```diff
@@ -13,6 +13,9 @@ func init() {
 			obj.To.Kind = "Service"
 		},
 		func(obj *TLSConfig) {
+			if len(obj.Termination) == 0 {
+				obj.Termination = TLSTerminationEdge
```

Comment:
It doesn't matter what we want here - if we shipped an API that
accepted an empty value and behaved one way, then realized it need to
be set for clarity, a default value is exactly how we solve that.